docs/reference/synonyms/apis/synonyms-apis.asciidoc

```diff
@@ -0,0 +1,31 @@
+[[synonym-management-apis]]
+== Synonym Management APIs
```

Comment:
```suggestion
== Synonyms APIs
```
Plural to match API path. Removed "management" for consistency with other CRUD APIs ("management" is implied). This change would cascade through to a few other places that I didn't mark.

Reply:
Changing that, thanks Chris!